dist/ember-json-api.js

```diff
@@ -319,9 +373,9 @@ define("json-api-adapter",
        * Use "links" key
        */
       serializeHasMany: function(record, json, relationship) {
-        var attr = relationship.key;
-        var type = this.keyForRelationship(relationship.type.typeKey);
-        var key = this.keyForRelationship(attr);
+        var attr = relationship.key,
+          type = this.keyForRelationship(relationship.type.typeKey),
+          key = this.keyForRelationship(attr);
```

Comment:
I used this style extensively, as it's a best practice from [Effective JavaScript](http://effectivejs.com/). That said, I agree that it could difficult to debug sometimes.